### ISSUE TITLE
Parquet: Row Group repeated for every row

### ISSUE THREAD

**Issue by dipeshkd** [slingdata-io/sling-cli#31]
Hi, Row Groups in Parquet are being repeated for each row on the file. I believe this should be on shown once, regardless of the number of rows in parquet file.

Eg. Below is the first 2 Row Groups. I chose file_max_rows: 10000 and so below is repeated 10000x.

``` json
  "RowGroups": [
    {
      "Ordinal": null,
      "NumRows": 1,
      "SortingColumns": [],
      "FileOffset": null,
      "TotalByteSize": 15460,
      "TotalCompressedSize": 12614
    },
    {
      "Ordinal": null,
      "NumRows": 1,
      "SortingColumns": [],
      "FileOffset": null,
      "TotalByteSize": 15460,
      "TotalCompressedSize": 12614
    },
   ...
]
```
Parquet is written to S3 bucket. Let me know if you need any additional information.

**Comment by flarco** [comment]
Hi, thanks, will look into this.
Sling uses https://github.com/fraugster/parquet-go, so not too sure what's going on yet.

**Comment by flarco** [comment]
@dipeshkd just released a new version (`1.0.47`). Should now show 1 Row Group.
